zerver/lib/actions.py

```diff
@@ -5680,3 +5681,16 @@ def notify_realm_export(user_profile: UserProfile) -> None:
     event = dict(type='realm_export',
                  exports=get_realm_exports_serialized(user_profile))
     send_event(user_profile.realm, event, [user_profile.id])
+
+def do_delete_realm_export(user_profile: UserProfile, export: RealmAuditLog) -> None:
+    # It's not actually possible for extra_data to be None at this point.
+    # However, mypy throws: "AnyStr" of "loads" cannot be "Optional[str]`
+    # if we don't check the None case.
+    export_extra_data = export.extra_data
+    if export_extra_data is not None:
```

Comment:
I would do `assert export_extra_data is not None` then, not add a useless and confusing if statement.

Reply:
Ahh of course, that should have been obvious. Fixed.